mod-data-import-converter-storage-server/src/main/java/org/folio/services/AbstractProfileService.java

```diff
@@ -24,6 +27,26 @@
   private static final Logger logger = LoggerFactory.getLogger(AbstractProfileService.class);
   private static final String GET_USER_URL = "/users?query=id==";
 
+  protected static final String INVENTORY_HOLDINGS = "INVENTORY_HOLDINGS";
+  protected static final String INVENTORY_INSTANCE = "INVENTORY_INSTANCE";
+  protected static final String INVENTORY_ITEM = "INVENTORY_ITEM";
+  protected static final String INVOICE = "INVOICE";
+  protected static final String MARC_AUTHORITY_RECORD = "MARC_AUTHORITY_RECORD";
+  protected static final String MARC_BIB_RECORD = "MARC_BIB_RECORD";
+  protected static final String MARC_HOLDINGS_RECORD = "MARC_HOLDINGS_RECORD";
+  protected static final String ORDER = "ORDER";
+  protected static final String ERROR = "ERROR";
```

Comment:
Better make an enum class

Reply:
I'm more inclined to apply enum here, instead of using string constants